packages/lazy-images/webpack.config.prod.js

```diff
@@ -0,0 +1,12 @@
+const path = require("path");
```

Comment:
I wonder if we should leave this out of the package, and handle file minification from the Jetpack plugin instead?

Reply:
Agreed, let's do that and keep the build scripts out of the package.

Reply:
I have created a new webpack config for packages at the root of the repo and added a new command to the package.json. Hopefully this is closer to what it should be. The webpack config for packages can be extended later if required i guess?